README.md

```diff
@@ -314,32 +318,44 @@ Angle Regression
 * [ape](http://cran.r-project.org/web/packages/ape/index.html) - Analyses of Phylogenetics and Evolution.
 * [pheatmap](http://cran.r-project.org/web/packages/pheatmap/index.html) - Pretty heatmaps made easy.
 
+## Social Network Analysis
+*Packages to construct, analyze and visualize network data.*
+
+* [igraph](http://igraph.org/r/) - A collection of network analysis tools.
+* [network](https://cran.r-project.org/web/packages/network/index.html) and [sna](https://cran.r-project.org/web/packages/sna/index.html) - Basic tools to manipulate relational data in R.
+* [networkDynamic](https://cran.r-project.org/web/packages/networkDynamic/) - Support for dynamic, (inter)temporal networks.
+* [statnet](http://statnet.org/) - The project behind many R network analysis packages.
+* [ergm](https://cran.r-project.org/web/packages/ergm/index.html), [ergm.count](https://cran.r-project.org/web/packages/ergm.count/), [ergm.userterms](https://cran.r-project.org/web/packages/ergm.userterms/index.html) and [xergm](https://cran.r-project.org/web/packages/xergm/index.html) - Exponential random graph models in R.

```

Comment:
Works for me. The rest of the `ergm` package family is in the "Reverse" section of the [CRAN page](https://cran.r-project.org/web/packages/ergm/).